app/models/server/models/Users.js

```diff
@@ -43,6 +43,7 @@ export class Users extends Base {
 		this.tryEnsureIndex({ federation: 1 }, { sparse: true });
 		this.tryEnsureIndex({ isRemote: 1 }, { sparse: true });
 		this.tryEnsureIndex({ 'services.saml.inResponseTo': 1 });
+		this.tryEnsureIndex({ openBusinessHour: 1 }, { sparse: true });
```

Comment:
Should by `openBusinessHours`, right?
```suggestion
		this.tryEnsureIndex({ openBusinessHours: 1 }, { sparse: true });
```